### PR TITLE
add error dialog / handling

### PR DESCRIPTION
**Pull request by zsinnema**
see #28

### CODE REVIEW

File: applications/osb-portal/package.json [hunk 1]
```diff
@@ -19,41 +19,43 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.10.1",
```
Comment (filippomc) [inline applications/osb-portal/package.json:22]: why is material ui a dev dependency?

Comment (filippomc) [reply applications/osb-portal/package.json:22]: same for react, redux, etc

Comment (zsinnema) [reply applications/osb-portal/package.json:22]: because they are loaded as js in the index.html

Comment (filippomc) [reply applications/osb-portal/package.json:22]: now I remember, thanks

**Pull request merged**